lib/wab/impl/init.rb

```diff
@@ -47,71 +48,56 @@ def exist_check(path)
         end
         
         def write_ui_controllers(dir, types)
-          template = File.open("#{__dir__}/templates/ui_controller.rb.template") { |f| f.read }
-          File.open("#{dir}/ui_controller.rb", 'w') { |f|
-            rest_flows = ''
-            types.each { |type|
-              rest_flows << %|
-    add_flow(WAB::UI::RestFlow.new(shell,
-                                   {
-                                     kind: '#{type}',
-                                   }, ['$ref']))
-|
-            }
-            f.write(template % { rest_flows: rest_flows })
+          rest_flows = ''
+          types.each { |type|
+            rest_flows << %|
+    add_flow(UI::RestFlow.new(shell, { kind: '#{type}'}, ['$ref']))|
```

Comment:
It is better to have an expanded version so users can just add a line for every attribute. With the change they will have to reformat as immediately.